lib/hypervisor/hv_kvm/monitor.py

```diff
@@ -917,3 +889,46 @@ def _RemoveFdset(self, fdset):
       # succeeded, the whole hot-add action will fail and the runtime file will
       # not be updated which will make the instance non migrate-able
       logging.info("Removing fdset with id %s failed: %s", fdset, err)
+
+
+class QgaConnection(QemuMonitorSocket):
```

Comment:
Not entirely sure why this is here, AFAIK there would be nothing using this code right now, correct? Maybe this should be added as a separate commit.

Reply:
Yes, this is not currently used. In the future, this will be used to communicate with the guest agent via qmp.

Reply:
I'm fine with leaving the `QgaConnection()` class here, it still serves the purpose of the PRs description "communication with the QGA agent is now possible". Maybe next time split PRs more granular.

Reply:
I still think this shouldn't be here or at least there should be a design document that explains what we want to achieve with agent support.

Reply:
I moved the QgaConnection class in another branch, so this is out for now.